scripting/tf2attribute_support.sp

```diff
@@ -582,6 +582,10 @@ MRESReturn OnPlayerCanAirDashPre(int client) {
 }
 
 MRESReturn OnPlayerCalculateMaxSpeedPost(int client, Handle hReturn, Handle hParams) {
+	if (client < 1 || client > MaxClients || !IsClientInGame(client) || IsFakeClient(client)) {
```

Comment:
The `IsFakeClient` check passing here would make it so bots will never be able to use `mult_move_speed_when_active`.

Reply:
You right, I need to find another way.